marlowe-chain-sync/src/Language/Marlowe/Runtime/Cardano/Api.hs

```diff
@@ -99,14 +99,6 @@ fromCardanoScriptData = \case
   C.ScriptDataNumber i -> I i
   C.ScriptDataBytes bs -> B bs
 
-toCardanoMetadata :: Metadata -> C.TxMetadataValue
```

Comment:
Is there an equivalent capability that provides this functionality? or is this truly not needed?  Clients will need to both read and write metadata.